container/appmaster/runit/service/flink/run

```diff
@@ -73,13 +73,37 @@ function add_kerberos_configurations() {
   fi
 }
 
+function add_ha_configurations() {
+  if [[ "${FLINK_HA_ENABLED}" == true ]]; then
+    export FLINK_JAVA_OPTS="$FLINK_JAVA_OPTS -Dhigh-availability=ZOOKEEPER"
```

Comment:
Why did you need to set `high-availability` via `FLINK_JAVA_OPTS` as opposed to the config file?